skimage/draw/draw.py

```diff
@@ -8,18 +8,33 @@
                     _bezier_curve)
 
 
-def _ellipse_in_shape(shape, center, radiuses):
-    """Generate coordinates of points within ellipse bounded by shape."""
+def _ellipse_in_shape(shape, center, radiuses, orientation=0.):
+    """ Generate coordinates of points within ellipse bounded by shape.
+    see: http://www.maa.org/external_archive/joma/Volume8/Kalman/General.html
+
+    :param shape: (int, int) shape of image
+    :param center: (float, float) position of center
+    :param radiuses: (float, float) size of the two half axis
+    :param orientation: float, orientation in radians
+    :return: [int], [int] coordinates of ellipse
+    """
     r_lim, c_lim = np.ogrid[0:float(shape[0]), 0:float(shape[1])]
     r, c = center
     ry, rx = radiuses
-    distances = ((r_lim - r) / ry) ** 2 + ((c_lim - c) / rx) ** 2
+    sin_alpha, cos_alpha = np.sin(orientation), np.cos(orientation)
+    x, y = (r_lim - r), (c_lim - c)
+    distances = ((x * cos_alpha + y * sin_alpha) / ry) ** 2 \
+                + ((x * sin_alpha + y * cos_alpha) / rx) ** 2
     return np.nonzero(distances < 1)
 
 
-def ellipse(r, c, yradius, xradius, shape=None):
+def ellipse(r, c, yradius, xradius, shape=None, orientation=0.):
```

Comment:
`yradius` and `xradius` are already renamed in `master` branch.
@Borda please, do a rebase.